src/blockchains/utxo/utxo_worker.ts

```diff
@@ -26,9 +38,10 @@ class UtxoWorker extends BaseWorker {
     });
   }
 
-  async init() {
+  async init(exporter: Exporter) {
     const blockchainInfo = await this.sendRequestWithRetry('getblockchaininfo', []);
     this.lastConfirmedBlock = blockchainInfo.blocks - this.CONFIRMATIONS;
+    await exporter.initPartitioner((event) => event['height']);
```

Comment:
We are utilizing the partitioner logic for the UTXO blockchain. Records would be distributed per partition depending on block number. This should give us even distribution and also determinism where each record goes.